e2e-tests/quote/cypress/e2e/journeys/quote/tell-us-about-your-policy/tell-us-about-your-single-policy-validation.spec.js

```diff
@@ -85,20 +97,100 @@ context('Tell us about the policy you need page - form validation', () => {
     });
 
     it('should focus on inputs when clicking summary error message', () => {
-      // currency
+      // policy length
       partials.errorSummaryListItemLinks().eq(0).click();
+      tellUsAboutYourPolicyPage[POLICY_LENGTH].input().should('have.focus');
+
+      // currency
+      partials.errorSummaryListItemLinks().eq(1).click();
       tellUsAboutYourPolicyPage[CURRENCY].input().should('have.focus');
 
       // contract value
-      partials.errorSummaryListItemLinks().eq(1).click();
+      partials.errorSummaryListItemLinks().eq(2).click();
       tellUsAboutYourPolicyPage[CONTRACT_VALUE].input().should('have.focus');
 
       // perecentage of cover
-      partials.errorSummaryListItemLinks().eq(2).click();
+      partials.errorSummaryListItemLinks().eq(3).click();
       tellUsAboutYourPolicyPage[PERCENTAGE_OF_COVER].input().should('have.focus');
     });
   });
 
+  describe('when `policy length` has a non-numeric value', () => {
+    it('should render a validation error', () => {
+      cy.navigateToUrl(url);
+
+      cy.keyboardInput(tellUsAboutYourPolicyPage[POLICY_LENGTH].input(), 'a');
+      submitButton().click();
+
+      cy.checkText(
```

Comment:
is it worth using the error cy command here? to make it 1 line?

Reply:
Makes sense, but this is in a lot of other places and we have a ticket to use more cypress commands for this kind of thing. I think we  should tackle it there. In this PR, just moved these assertions from one place to another